packages/amplify-category-notifications/src/notifications-backend-cfg-api.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ *  API to update Notifications category state in the state-db ( backend-config, frontend-config, teams-provider, amplify-meta)
+ */
+import {
+  $TSAny, pathManager, stateManager, AmplifySupportedService, AmplifyCategories, $TSContext,
+} from 'amplify-cli-core';
+import { INotificationsResourceBackendConfig, INotificationsResourceBackendConfigValue } from './notifications-backend-cfg-types';
+
+/**
+ * Persistent state management class for Notifications.
+ * Updates amplify-meta.json, backend-config.json
+ * TBD: updates to amplify-exports.js, cli-inputs.json
+ */
+export class NotificationsCfg {
```

Comment:
looks like these methods are all static too. perhaps just export functions from the module?